### DISCUSSION TITLE
安装出现问题

### DISCUSSION THREAD

**Discussion by Kemiaojun** [ywmoyue/biliuwp-lite#639] [Q&A]
### Bug 描述

<img width="867" alt="Image_1716285551333" src="https://github.com/ywmoyue/biliuwp-lite/assets/166591530/c753e7eb-a9c8-428b-8d41-b2bf61b4f8cf">
如图，第一步代码得到的是可以的

### 操作步骤

```plain text
按照步骤做的
```

### 期望结果

能成功安装

### 截图

<img width="867" alt="Image_1716285551333" src="https://github.com/ywmoyue/biliuwp-lite/assets/166591530/9df1a511-73f8-421c-8b69-5a9841e52a12">

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows11 23H2

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by GD-Slime** [comment]
根据图片中红字进行操作即可。需要用管理员模式打开Powershell再运行这条指令。

**Comment by ywmoyue** [comment]
![image](https://github.com/ywmoyue/biliuwp-lite/assets/28388300/8b22fac4-2ee7-4d8c-996d-170475206eb0)

该命令需要使用管理员身份执行

**Comment by ywmoyue** [comment]
@axiuluoshiwo 是否解决了问题呢
